### PR TITLE
Add Git Guideline

### PR DESCRIPTION
**Pull request by sbswarna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +15,19 @@ cd ..
 ls
 ls -a
 
+git add .
+$ git commit -m <commit msg>
+
+$ git fetch
+$ git rebase origin/develop
+
+$ git push --force-with-lease origin <your branch>
```
Comment (UitsHabib) [inline README.md:24]: I don't get the word "--force-with-lease". what is this shatabdi?

File: README.md [hunk 2]
```diff
@@ -15,4 +15,19 @@ cd ..
...
+
+$ git pull --rebase origin develop
```
Comment (UitsHabib) [inline README.md:26]: remove this line. we don't need it.

**Review by UitsHabib — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sbswarna) [reply README.md:26]: This is fine. We should not remove it.

File: README.md (hunk 1 above)
Comment (sbswarna) [reply README.md:24]: This is an interesting stuff.

**Review by UitsHabib — APPROVED** [review]
Nice work shatabdi. Great.

**Pull request merged**